examples/playground/vite.config.ts

```diff
@@ -12,8 +12,11 @@ export default defineConfig({
   },
   plugins: [react()],
   resolve: {
-    alias: {
-      ':': resolve(__dirname, './src'),
-    },
+    alias: [
```

Comment:
curious: why did you change to `find` syntax ?

Reply:
In fact it is a debug residue. I add some issues during build and find an issue related to this setting and I tried the verbose syntax, but both syntax are currently working. Do you prefer I rollback this change ?